at-hoangtran/Week3/Day4/Exercise4-Day4.rb

```diff
@@ -0,0 +1,32 @@
+class StringFomat
+  def human_name(name)
+    if name.is_a?(String)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/b841bab6cd381fd2aea25772a4ea14c33b24929d/at-hoangtran/Week3/Day4/Exercise4-Day4.rb#L3"></span>Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. (https://github.com/rubocop-hq/ruby-style-guide#no-nested-conditionals)</td>
    </tr>
  </tbody>
</table>